com.unity.cinemachine/Editor/Editors/InputAxisControllerEditor.cs

```diff
@@ -20,12 +20,18 @@ public class InputAxisControllerEditor : UnityEditor.Editor
         public override VisualElement CreateInspectorGUI()
         {
             var ux = new VisualElement();
-
-            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.PlayerIndex)));
-            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.AutoEnableInputs)));
-            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.ScanRecursively)));
-            var suppressInput = ux.AddChild(
-                new PropertyField(serializedObject.FindProperty(() => Target.SuppressInputWhileBlending)));
+            
+            SerializedProperty prop = serializedObject.GetIterator();
```

Comment:
I see what you're doing here, but for classes that are meant to be inherited, it's better to use PropertyDrawers if we can,  that way the user doesn't have to inherit the editor.  I will push a version using that strategy.

Reply:
I found a way to do this with PropertyDrawers, but I had to hack to work around a ListView bug, so this will have to be changed when they fix it.  I tagged all the relevant hacks with  `LISTVIEW_BUG_WORKAROUND`.

With this way, proper inspector drawing is automatic even with the default inspector for the inherited classes, so client doesn't have to know anything about our inspector.